cmd/upgrade.go

```diff
@@ -23,15 +23,15 @@ this version.`,
 	Args: cobra.NoArgs,
 	RunE: func(cmd *cobra.Command, _ []string) error {
 		flags := cmd.Flags()
-		oldDB, err := getString(flags, "old.database")
+		oldDB, err := flags.GetString("old.database")
 		if err != nil {
 			return err
 		}
-		oldConf, err := getString(flags, "old.config")
+		oldConf, err := flags.GetString("old.config")
 		if err != nil {
 			return err
 		}
-		db, err := getString(flags, "database")
+		db, err := flags.GetString("database")
 		if err != nil {
 			return err
 		}
```

Comment:
ℹ️ This is the only public command were we're not using `python` and therefore the env variables would **not** work. See https://github.com/filebrowser/filebrowser/pull/5550